libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -75,7 +76,8 @@ proc init*(_: type[GossipSubParams]): GossipSubParams =
       behaviourPenaltyDecay: 0.999,
       disconnectBadPeers: false,
       enablePX: false,
-      bandwidthEstimatebps: 100_000_000 # 100 Mbps or 12.5 MBps
+      bandwidthEstimatebps: 100_000_000, # 100 Mbps or 12.5 MBps
+      uselessAppBytesRateConfOpt: Opt.none(tuple[bytes: int, interval: Duration])
```

Comment:
We need a better name for this, overheadRateLimit? Not very pleased by this one either

Reply:
nonEssentialByteRateLimiter?

Reply:
These are not non essential though, it's more like overhead

Reply:
Used `overheadRateLimit`.